usefull-test-data/zipcodes.md

```diff
@@ -0,0 +1,134 @@
+usefull addresses
```

Comment:
Can we maybe put this in another folder? Seems weird to have a `usefull-test-data` folder in the root. Maybe we can name it just `docs/`?

Reply:
Good point, I agree